### DISCUSSION TITLE
OCT doesn't seem to be working

### DISCUSSION THREAD

**Discussion by officialmotive** [DISTRHO/Cardinal#362] [Q&A] [answered]
When I hook up the output of the octave control to the OCT module it stays the same key. It will change full octaves but will only play a single note continuously.

**Comment by dromer** [comment, marked as answer]
It seems `v/oct` and `shift` labels are reversed. I think this is just an issue with the panel.
When you mouse-over on the sockets you see their real functions.

**Reply by officialmotive** [reply]
Ah, that explains it, thank you!
